src/libCli/Call.cpp

```diff
@@ -146,7 +147,31 @@ namespace cli
         // Prepare the RPC call:
         std::multimap<grpc::string, grpc::string> clientMetadata;
         std::string methodStr = "/" + serviceName + "/" + methodName;
-        grpc::testing::CliCall call(channel, methodStr, clientMetadata);
+
+        // Get deadline for RPC from input or use custom
+        std::optional<std::chrono::time_point<std::chrono::system_clock>> timeout;
+        std::chrono::time_point<std::chrono::system_clock> defaultTimeout = std::chrono::system_clock::now() + std::chrono::milliseconds(100);
```

Comment:
be consistent in naming: `chrono::time_point`s are `deadlines` and `chrono::durations` are `timeouts`